chore: add bug template [skip ci] (#1858)
MidLevel/MLAPI: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: stat:awaiting triage, type:bug
assignees: ''

---

> **Describe the bug**
> A clear and concise description of what the bug is.

Reviewer: ```suggestion
**Describe the bug**

A clear and concise description of what the bug is.
```

> **To Reproduce**
> Steps to reproduce the behavior:
> 1. Go to '...'
> 2. Click on '....'
> 3. Scroll down to '....'
> 4. See error

Reviewer: ```suggestion
**To Reproduce**

Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

> **Describe the bug**
> A clear and concise description of what the bug is.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 1. Go to '...'
> 2. Click on '....'
> 3. Scroll down to '....'
> 4. See error
> 
> **Actual outcome**
> A clear and concise description of what broke.
> 
> **Expected outcome**
> A clear and concise description of what you expected to happen.
> 
> **Screenshots**
> If applicable, add screenshots to help explain your problem.
> 
> **Environment (please complete the following information):**
>  - OS: [e.g. Windows 10]
>  - Unity Version: [e.g. 2019.1]
>  - Netcode Version: [e.g. v6.0.1]
>  - Netcode Commit: [e.g. https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/commit/c102935df1d7e0928283b48948fe96e5d96dd961]
> 
> **Additional context**
> Add any other context about the problem here. Logs, code snippets would be useful here but please also consider attaching a minimal Unity project that reproduces the issue.
> Attach projects for reproduction if possible.

Reviewer 2: ```suggestion
### Description

A clear and concise description of what the bug is.

### Reproduce Steps

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

### Actual Outcome

A clear and concise description of what happened and broke.

### Expected Outcome

A clear and concise description of what you expected to happen.

### Screenshots

If applicable, add screenshots to help explain your problem.

### Environment

- OS: [e.g. macOS Monterey]
- Unity Version: [e.g. 2020.3]
- Netcode Version: [e.g. 1.0.0-pre.6]
- Netcode Commit: [e.g. https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/commit/ba418fa5b600ad9eb61fab0575f12fbecc2c6520]

### Additional Context

Add any other context about the problem here. Logs, code snippets would be useful here but please also consider attaching a minimal Unity project that reproduces the issue.
```
